source/viz.md

```diff
@@ -1554,13 +1554,14 @@ We also reduce the height of each chart
 so that they all fit in the same view.
 Note that we are re-using the chart we created just above,
 instead of re-creating the same chart from scratch.
+We also explicitly specify that `facet` is a categorical variable
+since faceting should only be done with categorical variables.
 
 ```{code-cell} ipython3
-
 morley_hist_facet = morley_hist_categorical.properties(
     height=100
 ).facet(
-    "Expt",
+    "Expt:N",
```

Comment:
This is to avoid the warning. It is technically possible to use a non-categorical variable with facet, which is why I chose the word "should".